source/unit.js

```diff
@@ -583,7 +588,107 @@ export class Unit {
     return massAmt / massUnit.magnitude_ ;
   }
 
+  /**
+   * Converts eq to mass.
+   * The calculation is done in two steps:
+   * 1. Calculate the equivalent mass of the substance by dividing its molecular weight by its valence.
+   *    For example, Ca2+ has a valence of 2 and a molecular weight of 40.08 g/mole, so its equivalent mass is 40.08/2 = 20.04 g/equiv.
+   * 2. Calculate the mass by dividing molecular mass by valence.
+   * 
+   * "The equivalent weight of an element is its gram atomic weight divided by its valence (combining power)."
+   * https://www.britannica.com/science/equivalent-weight
+   *
+   * @param {number} equivalents - The amount in equivalents to be converted.
+   * @param {object} targetUnit - The target/to unit for which the converted number is wanted.
+   * @param {number} molecularWeight - The molecular weight of the substance for which the conversion is being made.
+   * @param {number} valence - The valence of the substance for which the conversion is being made.
+   * @returns {number} - The equivalent mass in the specified mass unit.
+   */
+  convertEqToMass(equivalents, targetUnit, molecularWeight, valence) {
```

Comment:
I think this function needs a check like isMoleMassCommensurable (but for Mass & Eq) make sure the rest of the unit is commensurable before proceeding.

Reply:
I've implemented `isEqMassCommensurable`. To stay consistent with how `isMoleMassCommensurable` is used, I've added the check using `isMoleMassCommensurable` to the caller convertUnitTo instead of inside convertEqToMass